src/App.jsx

```diff
@@ -1,10 +1,64 @@
 import './App.css';
+import axios from "axios";
+import {useState} from "react";
 
 function App() {
 
+    const [countries, setCountries] = useState([]);
+
+    async function fetchCountries() {
+        try {
+            // cca3 (landcode) ook meegenomen om als key te gebruiken
+            const result = await axios.get(`https://restcountries.com/v3.1/all?fields=name,flag,population,region,cca3`);
+            setCountries(result.data.sort((countryA, countryB) => countryA.population - countryB.population));
+        } catch (e) {
+            console.error(e);
+        }
+    }
+
+    function changeCountryTextColor(country) {
+        let color = '';
+        switch (country.region) {
+            case 'Africa':
+                color = 'blue';
+                break;
+            case 'Americas':
+                color = 'green';
+                break;
+            case 'Asia':
+                color = 'red';
+                break;
+            case 'Europe':
+                color = 'yellow';
+                break;
+            case 'Oceania':
+                color = 'purple';
+                break;
+        }
+        return color
+    }
+
     return (
         <>
-            Maak je applicatie hier!
+            {/* Als countries[0] niet gevuld is, laat dan button zien */}
+            {!countries[0] && <button type="button" onClick={fetchCountries}
+            >Haal landen op!
+            </button>}
+
+            {countries.map((country) => {
+                return <div key={country.cca3}>
+                    {/*{console.log(country.cca3)}*/}
+                    <p className={changeCountryTextColor(country)}>
+                        {country.flag} {country.name.common}
+                    </p>
+                    <p>
+                        Has a population of {country.population}
+                    </p>
+                    <p>
+                        {country.region}
+                    </p>
+                </div>
```

Comment:
Je loopt door een array wanneer je de map methode gebruikt, vervolgens retourneer je informatie die je wilt gebruiken uit de array. Deze informatie wordt dus bij iedere iteratie in een bepaald element geplaatst, in dit geval in een div. Hierdoor creeër je div na div na div. Het is semantisch gezien beter om een ul element te plaatsen voordat je begint met de map methode, om vervolgens bij het retourneren na de map methode de informatie in een li element te plaatsen